wallets/rn_cli_wallet/src/components/Tag.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
+interface ITagProps {
+  value: string;
+  grey?: boolean;
+}
+
+export function Tag({value, grey}: ITagProps) {
+  return (
+    <View style={grey ? styles.greyTagContainer : styles.tagContainer}>
+      <Text style={grey ? styles.greyMainText : styles.mainText}>{value}</Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  tagContainer: {
+    backgroundColor: 'rgba(221, 241, 248, 1)',
+    minHeight: 26,
+    paddingHorizontal: 8,
+    paddingTop: 4,
+    borderRadius: 28,
+    marginRight: 4,
+    marginBottom: 8,
+  },
+  greyTagContainer: {
+    backgroundColor: 'rgba(60, 60, 67, 0.33)',
+    minHeight: 26,
+    paddingHorizontal: 8,
+    paddingTop: 4,
+    borderRadius: 28,
+    marginRight: 10,
+    marginBottom: 8,
+  },
```

Comment:
The only difference between some styles is the color, you could do something like this:

```
  tagContainer: {
    backgroundColor: 'rgba(221, 241, 248, 1)',
    minHeight: 26,
    paddingHorizontal: 8,
    paddingTop: 4,
    borderRadius: 28,
    marginRight: 4,
    marginBottom: 8,
  },
  greyTagContainer: {
    backgroundColor: 'rgba(60, 60, 67, 0.33)',
  },
```

and `<View style={[styles.tagContainer, grey && styles.greyTagContainer]}>`

Reply:
same comment for other similar styles